orchestrator/src/main/java/com/smoothstack/orchestrator/controller/UserController.java

```diff
@@ -46,11 +47,11 @@ String signIn() {
     }
 
     @GetMapping("/confirm")
-    String confirmMail(@RequestParam("token") String token) {
+    ResponseEntity<String> confirmMail(@RequestParam("token") String token) {
         System.out.println(token);
         Optional<ConfirmationToken> optionalConfirmationToken = confirmationTokenService.findConfirmationTokenByToken(token);
         optionalConfirmationToken.ifPresent(userService::confirmUser);
-        return token;
+        return ResponseEntity.status(HttpStatus.OK).body("Email confirmation successfull");
```

Comment:
Should only return this if the `optionalConfirmationToken` is present, otherwise return error message.

Reply:
made requested changes.